packages/app-form-builder/src/admin/views/Editor.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import HTML5Backend from "react-dnd-html5-backend";
+import { HTML5Backend } from "react-dnd-html5-backend";
```

Comment:
I guess this is the proper way of doing it? Just asking in case this was done by accident, sorry.

Reply:
Yes, API of the library changed. I made a slight upgrade of the library, because it wasn't going well with other libs. It's not the latest one though, because in latest there are some breaking changes to the API.

Reply:
Cool.